package.json

```diff
@@ -9,22 +9,23 @@
         "url": "http://github.com/amqp/rhea"
     },
     "dependencies": {
-        "debug": "0.8.0 - 3.5.0"
+        "debug": "4.1.1",
+        "jaeger-client": "^3.17.1"
```

Comment:
I would really like to avoid a fixed dependency here for everyone, regardless of whether they use tracing or not.